scripts/install.sh

```diff
@@ -17,59 +17,69 @@
 set -euv
 set -o pipefail
 
+## Run first ------------------------------------------------------------------
+# NOTE(cloudnull): Install the minimum packages required before anything can be
+#                  executed. While these should have been included in the base
+#                  kick, if they were not, they will need to be installed prior
+#                  to doing anything else.
+apt-get update
+DEBIAN_FRONTEND=noninteractive apt-get install -y \
+                                       python \
+                                       python-yaml \
+                                       python-virtualenv
```

Comment:
This is still installing packages which do not match the packages that were tested with the release, so it does not actually solve the problem at all. The amount of packages installed are reduced, sure, but it is still using packages that are not from the artifacted repo.

Reply:
Untill we can install an OS using RPC-Repo this will always be true.